.github/workflows/integtest.yaml

```diff
@@ -57,6 +57,24 @@ jobs:
       - name: tsc
         run: npx tsc
         working-directory: ${{ matrix.app_to_test }}
+      - name: Test redis cluster
+        if: matrix.app_to_test == 'ts-redis-cluster'
+        uses: vishnudxb/redis-cluster@1.0.5
+        with:
+          master1-port: 8000
+          master2-port: 8001
+          master3-port: 8002
+          slave1-port: 8003
+          slave2-port: 8004
+          slave3-port: 8005
+           # Running Test
+      - name: ping redis cluster
+        if: matrix.app_to_test == 'ts-redis-cluster'
+        run: |
+         sudo apt-get install -y redis-tools
```

Comment:
formatting: rest of file is indented with two spaces (this is one)

Reply:
will fix